crates/biome_json_parser/tests/json_test_suite/err/object_unterminated-value.json.snap

```diff
@@ -15,20 +14,26 @@ expression: snapshot
 ```
 JsonRoot {
     bom_token: missing (optional),
-    value: JsonObjectValue {
-        l_curly_token: L_CURLY@0..1 "{" [] [],
-        json_member_list: JsonMemberList [
-            JsonMember {
-                name: JsonMemberName {
-                    value_token: JSON_STRING_LITERAL@1..4 "\"a\"" [] [],
-                },
-                colon_token: COLON@4..5 ":" [] [],
-                value: JsonStringValue {
-                    value_token: JSON_STRING_LITERAL@5..7 "\"a" [] [],
-                },
+    value: JsonBogusValue {
```

Comment:
Is the whole node marked as a bogus value expected? Will a single key with unterminated string literals invalidate the whole root into a bogus value? This seems not the case when in arrays.

Same for other similar test cases.

Reply:
It was a flaw of the initial implementation. I've just pushed a second commit that should fix it. Let me know if it is better :)